pkglistgen/tool.py

```diff
@@ -814,10 +818,9 @@ def update_and_solve_target(
                 locales_from=target_config.get('pkglistgen-locales-from')
             )
 
-            if not self.skip_productcompose:
+            if use_product_composer:
                 self.write_productcompose()
-
-            if product_dir:
+            else:
```

Comment:
group files aren't used by product-composer AFAIK but nevertheless they were still built in product-compose mode. I think it's nice to still have those around, but I'm fine either way. But still, this is a change of behaviour with what's currently on master. What do you think we should do here?

Reply:
I was wrong on this, re-reading the current code again and the *current* generated artifacts, this looks good already and the group files aren't actually generated already in productcomposer mode. We had .group files in Micro 6.0 but they have been cleaned up. So please ignore my previous comment, sorry for that.